Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN        bash /tmp/env-config.sh
 # This layer is populated with up-to-date files from
 # Calypso development.
 COPY       . /calypso/
-RUN        npm ci
+RUN        npm ci --unsafe-perm
```

Comment:
With this, I'd expect `node_modules` and any files generated by `prepare` —run on postinstall— to belong to `root`.

Later, `npm run build` will still "downgrade" to `nobody` and produce files that belong to `nobody`.

This all makes sense, but I don't understand why changing to `nobody` disables `postinstall` scripts 🤷‍♂️